tools/ARIAtools/extractProduct.py

```diff
@@ -753,6 +751,10 @@ def main(inps=None):
     else:
         demfile=None ; Latitude=None ; Longitude=None
 
+    # Load or download mask (if specified).
+    if inps.mask is not None:
+        inps.mask = prep_mask([[item for sublist in [list(set(d['amplitude'])) for d in standardproduct_info.products[1] if 'amplitude' in d] for item in sublist], [item for sublist in [list(set(d['pair_name'])) for d in standardproduct_info.products[1] if 'pair_name' in d] for item in sublist]], inps.mask, standardproduct_info.bbox_file, prods_TOTbbox, proj, amp_thresh=inps.amp_thresh, arrshape=arrshape, workdir=inps.workdir, outputFormat=inps.outputFormat, num_threads=inps.num_threads)
+
     # Extract user expected layers
     export_products(standardproduct_info.products[1], standardproduct_info.bbox_file, prods_TOTbbox, inps.layers, inps.rankedResampling, dem=demfile, lat=Latitude, lon=Longitude, mask=inps.mask, outDir=inps.workdir, outputFormat=inps.outputFormat, stitchMethodType='overlap', verbose=inps.verbose, num_threads=inps.num_threads, multilooking=inps.multilooking)
```

Comment:
While it looks like this wasn't added in during this particular PR, clearly the Latitude/Longitude inputs are not used. Perhaps they are just vestiges so they should be removed altogether, along with the prep_dem call where they are created

Reply:
Actually, now I see how they are used in the "finalize_metadata" function. I think it is fine to pass as so and cleaner to generated in the "prep_dem" function as is, however we should probably give different variable names in case there are conflicts with other python modules (e.g. how about lats/lons instead?)